idl/chromadb/proto/debug.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package chroma;
+
+import "google/protobuf/empty.proto";
+
+message GetInfoResponse {
+  string version = 1;
+}
+
+service Debug {
+  rpc GetInfo(google.protobuf.Empty) returns (GetInfoResponse) {}
```

Comment:
It this just so when testing manually you can check its up?

Reply:
I need a handler that returns a successful response without panicking. Alternatively I could add a `shoud_panic` option to the request for `TriggerPanic`, or call a normal rpc route like QueryVectors

Reply:
Got it, thats fine makes sense